overrides/default.nix

```diff
@@ -855,6 +855,19 @@ lib.composeManyExtensions [
         }
       );
 
+      hidapi = super.hidapi.overridePythonAttrs (
+        old: {
+          propagatedBuildInputs = old.propagatedBuildInputs or [ ] ++ [
+            pkgs.libusb1
+          ];
+          postPatch = lib.optionalString stdenv.isLinux ''
+            libusb=${pkgs.libusb1.dev}/include/libusb-1.0
+            test -d $libusb || { echo "ERROR: $libusb doesn't exist, please update/fix this build expression."; exit 1; }
```

Comment:
Is this test necessary?

Reply:
I copied this from nixpkgs:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/hidapi/default.nix

I'm guessing it just makes it a bit more clear where the failure is, if the include/libusb-1.0 directory is no longer there (perhaps due to a version bump)